flathub.json

```diff
@@ -0,0 +1,3 @@
+{
+  "only-arches": ["x86_64"]
```

Comment:
sad panda

Reply:
I know :/ Hopefully next version won't have such limits.

Reply:
what prevents building from source?

Reply:
I went for a binary version for now to make the game available on FlatHub without too much work. Building from source is probably doable, but i haven't looked into it yet.

I automated the build in the upstream GitHub project, using a Debian docker image with the following dependencies (nothing too exotic):

- gettext
- imagemagick
- make
- openjdk-8-jdk
- python3-pip
- python3-setuptools

Then pip is used to install some Python packages:

- Pillow
- pafx
- pypng

Then it downloads gradle, which then downloads libgdx and other Java dependencies. These dependencies are only x86_64 for now on Desktop, but I plan to enable arm64 for the next version. It's not done yet because it requires a libgdx update, and libgdx updates often causes breakage.